src/components/OverflowMenu/OverflowMenu-test.js

```diff
@@ -3,6 +3,10 @@ import OverflowMenu from '../OverflowMenu';
 import Icon from '../Icon';
 import { shallow, mount } from 'enzyme';
 
+window.requestAnimationFrame = cb => {
```

Comment:
Does this line not work for rAF? https://github.com/carbon-design-system/carbon-components-react/blob/master/config/polyfills.js#L20

Reply:
Oh didn’t notice we have had the same thing already! Thanks @joshblack for pointing it out!